src/types/rpc/transaction.ts

```diff
@@ -0,0 +1,208 @@
+import { IResponse } from './response'
+
+enum TransactionType {
+  SEND = 'send',
+  RECEIVE = 'receive',
+  ALL = 'all',
+}
+
+type IBaseAddAm = {
```

Comment:
Please add `T` as a prefix http://pastel.wiki/en/Developers/pastel-electron-wallet/coding-style-guide 

Please update for all files